server.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fastify = require('fastify')({ logger: true })
+
+const postItem = require('./handlers/postItem');
+const getAllItems = require('./handlers/getAllItems');
+const getCategories = require('./handlers/getCategories')
+
+fastify.post('/postItem', postItem);
```

Comment:
You can use only `/items` as an endpoint for posting according to REST

Reply:
The same applies to the get items. You can use only `/items` and the method will be different. In REST we have a convention for using the same URL for handling an object with different methods (URI = URL + Method)